front/components/members/InvitationModal.tsx

```diff
@@ -258,28 +259,43 @@ export function InviteEmailModal({
 export function EditInvitationModal({
   owner,
   invitation,
-  isOpen,
-  setOpen,
+  onClose,
 }: {
   owner: WorkspaceType;
   invitation: MembershipInvitationType;
-  isOpen: boolean;
-  setOpen: (open: boolean) => void;
+  onClose: () => void;
 }) {
+  const [selectedRole, setSelectedRole] = useState<ActiveRoleType>(
+    invitation.initialRole
+  );
   const sendNotification = useContext(SendNotificationsContext);
   const confirm = useContext(ConfirmContext);
 
   return (
-    <Modal
-      title="Edit Invitation"
-      isOpen={isOpen}
+    <ElementModal
```

Comment:
Switched from `Modal` to `ElementModal` for the sake of consistency with the members drawer.